pennylane/transforms/unbroadcast_expand.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module contains the tape expansion function for unbroadcasting a
+broadcasted tape into multiple tapes."""
+import pennylane as qml
+from pennylane.transforms.batch_transform import batch_transform
```

Comment:
```suggestion
from .batch_transform import batch_transform
```

Not entirely sure I remember, but I _think_ you want a relative import here?

Reply:
This was a legacy decision from when I had put the transform in a different file and there were cyclic imports :D Thanks for the catch!